tests/pr_test.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
 )
 
+const basicExampleTerraformDir = "examples/basic"
```

Comment:
I wonder if we need to run this example in the tests, as all the functionality of this example is covered by other tests anyway.